zulip_bots/zulip_bots/bots/wikipedia/wikipedia.py

```diff
@@ -31,10 +31,12 @@ def handle_message(self, message, bot_handler, state_handler):
         bot_handler.send_reply(message, bot_response)
 
     def get_bot_wiki_response(self, message, bot_handler):
-        help_text = 'Please enter your message after @mention-bot'
+        help_text = 'Please enter your message after @{1}{0}{1}'
         query = message['content']
         if query == '':
-            return help_text
+            name = bot_handler.bot_identity().name
+            pad = '' if name.find(' ') < 0 else '**'
```

Comment:
(The number loc changes required would have been fewer if there is a method that returns the "{1}{0}{1}" padded string in the lib.)